api/sheets/get_cohort_members.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Get Cohort Members
+nav_order: 8
+parent: Google Sheets
+grand_parent: API
+---
+
+# Get the values of a given property for a node.
+
+## =DCCOHORTMEMBERS(dcids)
+
+Given a [cohort](http://browser.datacommons.org/kg?dcid=Cohort) or a row of cohort DCIDs, get the [members](http://browser.datacommons.org/kg?dcid=member) of each cohort.
+
+**Arguments**
+*    `dcids` - cohort DCIDs to get the members of
+
+**Returns**
+
+The DCIDs of the cohort members. For a single DCID, the result is a column of members. For a row of DCIDs, the result is a matrix with each column being the members of the DCID at the column's index. For a column of DCIDs, the result is a matrix with each row being the members of the DCID at the row's index.
```

Comment:
this is repeated twice

Reply:
You mean it's there in get_property.md as well as in get_cohort_members.md? The two sentences in this paragraph should have "row" and "column" swapped.